src/panel/pets.ts

```diff
@@ -178,6 +179,8 @@ export function createPet(
             return new Cat(...standardPetArguments, PetSpeed.normal);
         case PetType.chicken:
             return new Chicken(...standardPetArguments, PetSpeed.normal);
+        case PetType.deno:
+            return new Deno(...standardPetArguments, PetSpeed.normal);
```

Comment:
I've been playing around with a couple of these and think the pet animations might look more natural at `slow` instead of `normal`. It would walk and run slower (fewer pixels/sec), but it is a big ol' dinosaur.

What do you think?

Reply:
awww yeah, could be kind of cute lumbering around

Reply:
Merging with "slow"